### PR TITLE
Adding log level config from env

### LINKED ISSUE #243: Enable logging globally and in production
I would like to enable logging and be able to set log level in a production environment. The current setup hs logging enabled but logs only default level. I suggest reading the log level from the environment at initialization time.

```
if os.environ['PSIJ_LOG_LEVEL']:
    logging.basicConfig(format='%(name)s:%(levelname)s:%(message)s',
                        level=os.environ['PSIJ_LOG_LEVEL'].upper())
logger = logging.getLogger(__name__)
```

### PR DESCRIPTION
**Pull request by wilke**
closes #243

### CODE REVIEW

File: src/psij/__init__.py [hunk 1]
```diff
@@ -26,8 +26,12 @@
     'InvalidJobException', 'UnreachableStateException', 'Export', 'Import'
 ]
 
+if os.environ['PSIJ_LOG_LEVEL']:
```
Comment (hategan) [inline src/psij/__init__.py:29]: If PSIJ_LOG_LEVEL is not defined, that line will throw an exception and prevent any code using psij from working.

I think what you want to check is `if 'PSIJ_LOG_LEVEL' in os.environ`:

Comment (wilke) [reply src/psij/__init__.py]: Changed, thanks for the suggestion

**Comment by hategan** [comment]
Do we maybe want to provide a more comprehensive solution, such as `PSIJ_LOG_CONFIG` that points to a file which would allow users to configure all logging aspects?

That said, PSI/J is a library. It is not meant to be run in a standalone fashion. In particular, since logging configuration is global, any PSIJ_ logging configuration options would, in fact, apply to all python code that uses psij and may lead to conflicts if user code configures different options. 

We should perhaps discuss this, but it seems like logging configuration should be left to the main application that uses PSI/J.

**Comment by kylechard** [comment]
Discussed on call and agreed this should be moved to the CLI.

**Comment by hategan** [comment]
Moving to 1.0.0

**Comment by hategan** [comment]
Changing this to draft until it gets moved to the CLI and it's ready for review

**Comment by hategan** [comment]
I'm closing this since there has been no activity on it for over 6 months.

**Pull request closed without merging**